composer.json

```diff
@@ -17,7 +17,7 @@
   ],
   "require": {
     "php": ">=5.4",
-    "moxie-lean/wp-endpoint": "1.*.*",
+    "moxie-lean/wp-endpoint": "2.*.*",
```

Comment:
I agree, this is the standard way of doing it. The problem is we already have several packages using the * way... so using ^ here will make this package inconsistent and (if i remember rightly) causes issues between them.

We can open a task to go through all active packages to fix this.